common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -277,7 +277,7 @@ public MicrosoftStsAccount createAccount(
         }
 
 
-        builder.setLibraryName(DiagnosticContext.getRequestContext().get(AuthenticationConstants.SdkPlatformFields.PRODUCT));
+        builder.setLibraryName(DiagnosticContext.INSTANCE.getRequestContext().get(AuthenticationConstants.SdkPlatformFields.PRODUCT));
```

Comment:
Did we make change to DiagnosticContext in this PR? If not how was this working until now?

Reply:
DiagContext's change was made in the Logger PR.

Reply:
Right. So I believe we are in a state right now where both with and without the `INSTANCE` would work?

Reply:
Yes. I'll eventually retire the old methods though. It's currently kept to avoid making a breaking change.